spec/verifyUserEmails.spec.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
```

Comment:
I assume we already have a spec file for email verification? If so, please move the tests to there instead of creating a new file.

Reply:
Yes, please move the whole `describe` group there, you already named it well. And then rename the file from `EmailVerificationToken.spec.js` to `EmailVerification.spec.js`.